zilliqa/src/cfg.rs

```diff
@@ -517,6 +520,39 @@ impl Default for ConsensusConfig {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize, Debug)]
+#[serde(deny_unknown_fields)]
+pub struct TxnPoolConfig {
+    /// Maximum number of transactions transaction pool can hold
+    pub maximum_global_size: u64,
+    /// Maximum number of transactions per single sender
+    pub maximum_txn_count_per_sender: u64,
+    /// total slots for all senders
```

Comment:
What is a 'slot'?

Reply:
From the impl it looks like this is a limit on the number of unique senders in the pool.

Why is that necessary when we already have the `maximum_global_size`?